docs/using_edge_services/policy_user_cases.md

```diff
@@ -60,6 +60,27 @@ Administrators can create a deployment policy, and {{site.data.keyword.edge_notm
 
 Because deployment policies capture the more dynamic, business-like service properties and constraints, they are expected to change more often than service policy. Their lifecycle is independent from the service they refer to, which gives the policy administrator the ability to state a specific service version or a version range. {{site.data.keyword.edge_notm}} then merges service policy and deployment policy, and then attempts to find nodes whose policy is compatible with that.
 
+## How do you update the constraints of an existing deployment policy with the hzn cli?
```

Comment:
Simplify title to "Deployment policy updating"

Reply:
This additional information should be moved to the ## Constraints section.